### DISCUSSION TITLE
Someone take my code to made this public from his account

### DISCUSSION THREAD

**Discussion by dheerajgoel44** [community/community#41074] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

If some one please help me, as someone took my code and made this public on github from his account, how i can remove this as public

**Comment by dheerajgoel44** [comment]
if someone please help in this reagrds?

**Comment by DSDmark** [comment, marked as answer]
Hi @dheerajgoel44,

in that case, you can report that person by click **block and abuse** button, which is just down below his person profile. Like this.

![image](https://user-images.githubusercontent.com/85744816/206449564-d44a5808-69ee-45b8-9e03-849a590913d4.png)

for more details, read this :- https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @dheerajgoel44! The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
